presentation/dto/user.go

```diff
@@ -6,22 +6,19 @@ import (
 )
 
 type UserInputDTO struct {
-	ID       uuid.UUID `json:"id" `
-	Username string    `json:"username"`
-	Email    string    `json:"email"`
-	Password string    `json:"password"`
+	Email    string `json:"email"`
+	Password string `json:"password"`
```

Comment:
Validation is missing. Example can be found in credentials dto. Without validation it is possible to register without an email.
Also maybe check if email can be declared as required in the gorm entity creation.

Reply:
You are right, I made a new issue for that. I would like to handle it in another PR to be able to merge this one without further delay since we kind of need this for our frontend :)